vendor/auto-reload.js

```diff
@@ -32,6 +32,34 @@
 
       // Hack to force page repaint after 25ms.
       if (forceRepaint) setTimeout(function() { document.body.offsetHeight; }, 25);
+    },
+
+    javascript: function(){
+      console.log(1);
+      var scripts = [].slice.call(document.querySelectorAll('script'));
+      var textScripts = scripts.map(function(script) { return script.text }).filter(function(text) { return text.length > 0 });
+      var srcScripts = scripts.filter(function(script) { return script.src });
+
+      var loaded = 0;
+      var all = srcScripts.length;
+      var onLoad = function() {
+        loaded = loaded + 1;
+        if (loaded === all) {
+          if (typeof require === 'object' && require.reset) require.reset();
+          textScripts.forEach(function(script) { eval(script); });
+        }
+      }
```

Comment:
This is needed to re-evaluate the text scripts that enter the application (like `require('initialize');`)